docs/docs/production/callbacks/index.mdx

```diff
@@ -0,0 +1,55 @@
+import CodeBlock from "@theme/CodeBlock";
+
+# Events / Callbacks
+
+LangChain provides a callback system that allows you to hook into the various stages of your LLM application. This is useful for logging, [monitoring](../tracing), [streaming](../../modules/models/llms/additional_functionality#streaming-responses), and other tasks.
+
+You can subscribe to these events by using the `callbacks` argument available throughout the API. This method accepts a list of handler objects, which are expected to implement one or more of the methods described in the [API docs](../../api/callbacks/interfaces/CallbackHandlerMethods).
+
+## Dive deeper
+
+import DocCardList from "@theme/DocCardList";
+
+<DocCardList />
+
+## How to use callbacks
+
+The `callbacks` argument is available on most objects throughout the API (Chains, Models, Tools, Agents, etc.) in two different places:
+
+- **Constructor callbacks**: defined in the constructor, eg. `new LLMChain({ callbacks: [handler] })`, which will be used for all calls made on that object, and will be scoped to that object only, ie doen't affect any child object, eg. if you pass a handler to the LLMChain constructor, it will not be used by the Model attached to that chain.
```

Comment:
I think this is actually swapped. Constructor callbacks are scoped to that particular object and not passed through / inherited. Request callbacks are piped through all sub objects